docs/development.rst

```diff
@@ -726,97 +699,79 @@ This section covers all variants of new versions:
 * Starting a new update version (M.N.Unew) based on the stable branch of its
   minor version
 
-The description assumes that the ``pywbem/pywbem`` repo is cloned locally in a
-directory named ``pywbem``. Its upstream repo is assumed to have the remote name
-``origin``.
+This description assumes that you are authorized to push to the remote repo
+at https://github.com/pywbem/pywbem and that the remote repo
+has the remote name ``origin`` in your local clone.
 
 Any commands in the following steps are executed in the main directory of your
 local clone of the ``pywbem/pywbem`` Git repo.
 
-1.  Set shell variables for the version that is being started and the branch it
-    is based on:
-
-    * ``MNU`` - Full version M.N.U that is being started
-    * ``MN`` - Major and minor version M.N of that full version
-    * ``BRANCH`` -  Name of the branch the version that is being started is
-      based on
-
-    When starting a new major version (e.g. ``1.0.0``) based on the master
-    branch:
+1.  Create and push the start branch (replace M,N,U accordingly):
 
     .. code-block:: sh
 
-        MNU=1.0.0
-        MN=1.0
-        BRANCH=master
+        VERSION=M.N.U make start_branch
 
-    When starting a new minor version (e.g. ``0.9.0``) based on the master
+    This uses the default branch determined from ``VERSION``: For ``M.N.0``,
+    the ``master`` branch is used, otherwise the ``stable_M.N`` branch is used.
+    That covers for all cases except if you want to start a new minor version
+    based on an earlier stable branch. In that case, you need to specify that
     branch:
 
     .. code-block:: sh
 
-        MNU=0.9.0
-        MN=0.9
-        BRANCH=master
+        VERSION=M.N.0 BRANCH=stable_M.N make start_branch
 
-    When starting a new minor version (e.g. ``0.8.1``) based on the stable
-    branch of its minor version:
-
-    .. code-block:: sh
+    This includes the following steps:
 
-        MNU=0.8.1
-        MN=0.8
-        BRANCH=stable_${MN}
-
-2.  Create a topic branch for the version that is being started:
-
-    .. code-block:: sh
+    * create the start branch (``start_M.N.U``), if not yet existing
```

Comment:
Change "not yet existing" t "it does not exist"

Reply:
DONE, in both places.